xgi/classes/reportviews.py

```diff
@@ -194,8 +209,26 @@ def __repr__(self):
         """
         return f"{self.__class__.__name__}({tuple(self)})"
 
+    def __call__(self, size):
+        """Filter the results by size."""
+        bunch = [id for id in self._id_dict if len(self._id_dict[id]) == size]
+        return self.from_view(self, bunch)
+
+    @classmethod
+    def from_view(cls, view, bunch=None):
+        """Create a view from another view.
+
```

Comment:
Could you add the parameters in the docstring?

Reply:
Done! I'll write more extensive docs in the docs branch